packages/next-tinacms-cloudinary/package.json

```diff
@@ -13,33 +13,49 @@
   },
   "dependencies": {
     "cloudinary": "^1.25.2",
-    "lodash": "^4.17.21",
     "multer": "^1.4.2",
-    "next": "10.0.5",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-is": "^17.0.2",
-    "tinacms": "0.31.0",
-    "tsup": "^3.7.0",
-    "typescript": "^3.8.3"
+    "tsup": "^3.7.0"
   },
   "devDependencies": {
+    "@tinacms/core": "^0.31.0",
```

Comment:
I'm assuming you got yarn complaining about these, we had to do the same thing over [here](https://github.com/tinacms/tina-graphql-gateway/blob/main/packages/tina-graphql-gateway/package.json#L47-L53). Not ideal but looks good to me

Reply:
I removed them and did not get any errors.